ui/components/SourcesTable.tsx

```diff
@@ -109,12 +108,8 @@ function SourcesTable({ className, sources }: Props) {
                 return "-";
               case FluxObjectKind.KindHelmRepository:
                 text = (s as HelmRepository).url;
-                if (
-                  (s as HelmRepository).repositoryType != HelmRepositoryType.OCI
-                ) {
-                  url = text;
-                  link = true;
-                }
+                url = text;
+                link = true;
```

Comment:
Just want to make sure that SourcesTable is the only one out of AutomationsTable, EventsTable, ReconciledObjectsTable, heck even FluxRuntime where link changes need to be made